vars/buildDockerAndPublishImage.groovy

```diff
@@ -50,6 +52,7 @@ def call(String imageShortName, Map userConfig=[:]) {
     unstash: '', // Allow to unstash files if not empty
     dockerBakeFile: '', // Specify the path to a custom Docker Bake file to use instead of the default one
     dockerBakeTarget: 'default', // Specify the target of a custom Docker Bake file to work with
+    cacheTo: '', // New parameter for Docker build cache export using cache-to
```

Comment:
You need unit test to cover both the default case and with a string passed as argument